framework/Samplers/DynamicEventTree.py

```diff
@@ -954,4 +955,4 @@ def localInitialize(self):
       self.branchProbabilities[key] = [self.distDict[key].cdf(float(self.branchValues[key][index])) for index in range(len(self.branchValues[key]))]
     self.limit = sys.maxsize
     # add expected metadata
-    self.addMetaKeys(*['RAVEN_parentID','RAVEN_isEnding'])
+    self.addMetaKeys(*['RAVEN_parentID','RAVEN_isEnding',])
```

Comment:
What does the trailing comma do?

Reply:
left over...but it does not impact the addition of the metadata keys... removed